acct-user/logstash/logstash-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
```

Comment:
This is out of order.

Reply:
You mean the commit? It's strange, because locally this commit is the second one, hm..

Reply:
After another rebase and force push it seems fine now.